<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTempoDetect</class>
 <widget class="QDialog" name="DlgTempoDetect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkauTan: Tempo detector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblSongInfo">
     <property name="text">
      <string notr="true">SongInfo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stored MPM:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Manual:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMpmManual">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ID3 tag:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMpmId3">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMpmFilename">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Detection algorithm settings:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Level calculation algorithm:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwLevelAlgorithm">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Window size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwWindowSize">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Stride:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwStride">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLineEdit" name="leSampleRate">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Sample rate:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="btnSaveDebugLevels">
         <property name="text">
          <string>Save raw audio with Levels...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QPushButton" name="btnSaveDebugBeats">
         <property name="text">
          <string>Save raw audio with Beats...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Local maximum distance:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="leLocalMaxDistance">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="leMinTempo">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Max tempo:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="leMaxTempo">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Min tempo:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Level normalization window:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="leNormalizeLevelsWindowSize">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>History:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="twDetectionHistory">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>LevelAlg</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Window</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stride</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SampleRate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LocMaxDist</string>
      </property>
      <property name="toolTip">
       <string>Local maximum distance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MinTempo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MaxTempo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LvlNormWin</string>
      </property>
      <property name="toolTip">
       <string>Level normalization window size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tempo</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MPM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Confidence</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leMpmManual</tabstop>
  <tabstop>leMpmId3</tabstop>
  <tabstop>leMpmFilename</tabstop>
  <tabstop>lwLevelAlgorithm</tabstop>
  <tabstop>lwWindowSize</tabstop>
  <tabstop>lwStride</tabstop>
  <tabstop>leLocalMaxDistance</tabstop>
  <tabstop>leMinTempo</tabstop>
  <tabstop>leMaxTempo</tabstop>
  <tabstop>btnSaveDebugLevels</tabstop>
  <tabstop>btnSaveDebugBeats</tabstop>
  <tabstop>twDetectionHistory</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
